cdap-ui/app/hydrator/hydrator-modal.less

```diff
@@ -65,9 +64,9 @@
 
         // Adjust max-width of modal title when my-jump-link is present
         &.with-jump {
-          max-width: ~"calc(100% - 440px)";
-          max-width: ~"-moz-calc(100% - 440px)";
-          max-width: ~"-webkit-calc(100% - 440px)";
+          max-width: ~"calc(100% - 180px)";
+          max-width: ~"-moz-calc(100% - 180px)";
+          max-width: ~"-webkit-calc(100% - 180px)";
```

Comment:
Do we still need browser prefixes for `calc`?

Reply:
Good catch. We don't need it. We use autoprefixer in our gulp build and that should take care of it. Have removed.